src/TrueLayer/Mandates/MandatesApi.cs

```diff
@@ -65,7 +71,10 @@ public async Task<ApiResponse<CreateMandateResponse>> CreateMandate(CreateMandat
 
         //TODO: is it correct that this method expects a mandate type?
         /// <inheritdoc />
-        public async Task<ApiResponse<MandateDetailUnion>> GetMandate(string mandateId, MandateType mandateType, CancellationToken cancellationToken = default)
+        public async Task<ApiResponse<MandateDetailUnion>> GetMandate(
```

Comment:
One day it would be nice standardising the handling for scopes across libs. I think we should also start from the API and re-assess if using scopes to segregate type of mandates is correct... It does not seem to be very inline with Oauth?

Reply:
That's something we considered, it will be nice to remove those scopes